src/ipmanager/ui/views.py

```diff
@@ -28,7 +29,8 @@ class SingleGroupView(DetailView):
   
   def get_context_data(self, **kwargs):
     context = super().get_context_data(**kwargs)
-
+    form = RelationForm(initial={'subject' : self.object})
```

Comment:
These two lines can be collapsed into a single line, in and inline with the `context.update()` on lines 36-40:

```python
context.update(
    relation_form=RelationForm(initial={'subject' : self.object}),
    ip_ranges=...
)
```